client/my-sites/customer-home/cards/features/grow-earn/style.scss

```
@@ -0,0 +1,9 @@
+.grow-earn {
+    padding-bottom: 12px;
+
+	&__card-subheader {
```

Let's avoid parent selectors here too 🙂 
```suggestion
	.grow-earn__card-subheader {
```

---

9d52b0e28d